README.md

```diff
@@ -1,76 +1,81 @@
 # Azure Terraform
+[![Build Status](https://travis-ci.com/Azure/vscode-azureterraform.svg?branch=master)](https://travis-ci.com/Azure/vscode-azureterraform)
+[![Release Status](https://vsmarketplacebadge.apphb.com/version-short/ms-azuretools.vscode-azureterraform.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureterraform)
 
 The VSCode Azure Terraform extension is designed to increase developer productivity authoring, testing and using Terraform with Azure. The extension provides terraform command support, resource graph visualization and CloudShell integration inside VSCode.
 
 ![overview](images/overview.png)
```

Comment:
vscode will not render PNG/GIF with the relative path in MD. Please use the absolute path for these kinds of resources. Of course, you can wait and change it after the repo becomes public. The absolute path is like `https://raw.githubusercontent.com/Azure/vscode-azureterraform/master/image/overview.png`